tests/test_preview.py

```diff
@@ -250,3 +257,116 @@ def test_that_logo_files_exist(logo):
 def test_that_logos_must_have_at_least_one_file(partially_initialised_class):
     with pytest.raises(TypeError):
         partially_initialised_class()
+
+
+@pytest.mark.parametrize('filetype, sentence_count, page_number, expected_response_code', [
+    ('png', 10, 1, 200),
+    ('png', 10, 2, 400),
+    ('png', 50, 2, 200),
+    ('png', 50, 3, 400),
```

Comment:
why do we care about 400 responses here? surely this test should just be "make sure that if there is a value in the cache with the correct key, that value is returned and we don't generate a pdf"

Reply:
You are right I copied the whole test for cached but the 400s are already tested and the fact caching is turned on makes no difference to the flow. Removed Test.